server/db/config.js

```diff
@@ -1,5 +1,5 @@
-require('../env');
-// const data = require('./data.json');
+// require('../env');
+const data = require('./data.json');
```

Comment:
This is going to error our for all of us that don't have this useless file in the exact location that you have.